38-audio-classifier/README.md

```diff
@@ -0,0 +1,220 @@
+      - [Prebuilt Dataset](#prebuilt-dataset)
+      - [Custom Dataset](#custom-dataset)
+    - [Training the Model](#training-the-model)
+    - [Downloading the Model](#downloading-the-model)
+  - [Notehub Setup](#notehub-setup)
+    - [Environment Variables](#environment-variables)
+    - [`detections.qo`](#detections-qo)
+  - [Classifier Firmware](#firmware-setup)
+  - [Testing](#testing)
+  - [Blues Community](#blues-community)
+
+## Overview
+
+In this application, you'll build a binary audio classifier. "Binary" here means there are two classes of audio the model will be able to classify: the sound you're interested in and everything else. In this README, we'll use the example of a "running faucet" detector, leveraging [this audio recognition tutorial](https://docs.edgeimpulse.com/docs/tutorials/audio-classification) from Edge Impulse. However, you can easily apply the principals described here to any other binary audio classification task (we even built a cat meow detector!). You'll use Edge Impulse to collect data, train the model, and download the model as as Arduino library for use in your Swan firmware. The firmware continually collects 1 second clips of audio via the microphone and feeds them into your audio classifier model, publishing a note to Notehub if it detects that the faucet is running.
+
+## You Will Need
+
+### Hardware
+
+- [Notecard](https://shop.blues.io/collections/notecard/products/note-wbna-500)
+- [Notecarrier F](https://blues.io/products/notecarrier/notecarrier-f/)
+- [Swan](https://shop.blues.io/collections/swan/products/swan)
+- [Adafruit PDM MEMS Microphone Breakout](https://www.adafruit.com/product/3492)
+- Soldering iron and flux
+- Male-to-female jumper wires
+- Micro USB cable
+
+### Software
+
+- [Visual Studio Code (VS Code)](https://code.visualstudio.com/)
+- [PlatformIO extension for VS Code](https://platformio.org/install/ide?install=vscode)
+
+## Hardware Setup
+
+1. Assemble Notecard and Notecarrier as described [here](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/).
+2. Plug the Swan into the Notecarrier, aligning the Swan's male headers with the Notecarrier's female headers.
+3. Using your soldering iron and flux, solder 5 of the headers through the holes on the breakout:
+
+    ![Mic soldered](images/mic_soldered.jpg)
+
+4. Make the following connections between the Notecarrier and the mic breakout using the jumper wires:
+
+    | Notecarrier | Breakout |
+    | ----------- | ---------|
+    | F_3V3       | 3V       |
+    | GND         | GND      |
+    | F_A0        | CLK      |
+    | F_A2        | DAT      |
+
+    ![Hardware assembled](images/hardware_assembled.jpg)
+
+5. Connect the Swan to your development PC with the micro USB cable.
+
+## Building the Audio Classifier Model
+
+First, sign up for an [Edge Impulse](https://studio.edgeimpulse.com/) account and log in. Click "Create new project", enter a project name and click "Create new project".
```

Comment:
Again trivial, but I’d make this a quick bulleted list, even if it’s just 3 bullets. That’s a lot of steps on one line.

Reply:
> for an Edge Impulse account

for a free Edge Impulse account

You could consider mentioning this when you first mention Edge Impulse too, just so people realize they don’t have to sign up for an expensive service to try this out.